src/containers/buttons/ViewBidsButton/ViewBidsButton.tsx

```diff
@@ -0,0 +1,93 @@
+//- Containers Imports
+import { BidList } from 'containers';
+
+//- Library Imports
+import { Bid } from '@zero-tech/zns-sdk/lib/zAuction';
+import { Maybe, Metadata } from 'lib/types';
+import { Domain } from '@zero-tech/zns-sdk/lib/types';
+
+//- Components Imports
+import { FutureButton, Overlay, TextButton } from 'components';
+
+//- Constants Imports
+import { LABELS } from './ViewBidsButton.constants';
+
+interface ViewBidButtonProps {
+	bids?: Bid[];
+	domain?: Domain;
+	buttonText?: string;
+	className?: string;
+	isTextButton?: boolean;
+	refetch: () => void;
+	style?: React.CSSProperties;
+	highestBid?: number;
+	isLoading?: boolean;
+	domainMetadata?: Maybe<Metadata>;
+	setIsViewBidsOpen?: (state: boolean) => void;
+	isViewBidsOpen?: boolean;
+}
+
+const ViewBidsButton = ({
+	bids,
+	domain,
+	buttonText,
+	className,
+	isTextButton,
+	refetch,
+	style,
+	highestBid,
+	isLoading,
+	domainMetadata,
+	setIsViewBidsOpen,
+	isViewBidsOpen,
```

Comment:
Generally when I make overlay buttons I would define these internally, as the purpose of an overlay button is to encapsulate all the functionality of showing/hiding the modal. I don't think you'd need to control the modal rendering from outside of this component, but I could be wrong.